mep/common/forms.py

```diff
@@ -0,0 +1,56 @@
+from typing import Any, Mapping
+
+from django import forms
+from django.utils.text import mark_safe
+
+class CheckboxFieldset(forms.CheckboxSelectMultiple):
+
+    template_name = 'common/widgets/checkbox_fieldset.html'
+
+
+class FacetChoiceField(forms.MultipleChoiceField):
+    '''Add CheckboxSelectMultiple field with facets taken from solr query'''
+    # Borrowed from https://github.com/Princeton-CDH/derrida-django/blob/develop/derrida/books/forms.py
+    # customize multiple choice field for use with facets.
+    # no other adaptations needed
```

Comment:
Are these comments still valid? Is the only change the widget?

Reply:
Just the widget, I can add something about the custom `name` logic.